redbeams-api/src/main/java/com/sequenceiq/redbeams/api/model/describe/DescribeRedbeamsV4Response.java

```diff
@@ -13,7 +13,7 @@
 @ApiModel("DescribeRedbeamsV1Response")
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
-public class DescribeRedbeamsResponse {
+public class DescribeRedbeamsV4Response {
```

Comment:
I don't understand this classname at all, and additionally, it doesn't agree with the ApiModel tag.

Reply:
Maybe better = DatabaseServerAllocationOutcome ?

It might be handy, on successful creation, to include a DatabaseServerConfig object, so that the caller doesn't need to go get it.

Reply:
My bad on the API model tag -- FreeIPA labels everything V1, everything else seems to label it V4.

One thing is that we're not actually going to get the outcome immediately since it's not synchronous -- Describe may be the wrong verb here.

Reply:
I'm going to add this as DatabaseServerV4Response for now. If we need to grow it into something else, we can do so as needed.